packages/appeals-service-api/src/routes/documentMetadata.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { getDocumentMetadata } = require('../services/document-metadata.service');
+const router = express.Router();
+
+router.get('/:caseref', async (req, res) => {
```

Comment:
The only question mark here is that the current examples we have of appeals case ref values all include protected url characters that we might need som kind of parsing function for to share in different places. @bryn-pins Did we end up needing to do this with any of the other appeals list views on the links to the appeal details views, because I think we mentioned we might need it there as well?

Reply:
I used URL encoded values for my testing and it worked fine as is. The front end will need to urlencode the caseRef.